### ISSUE TITLE
(needs edits) Submission Status Review

### ISSUE THREAD

**Issue by mengA6** [department-of-veterans-affairs/va.gov-team#78180]
## Issue Description

From [OCTO prioritization spreadsheet](https://dvagov.sharepoint.com/:x:/r/sites/vaabdvro/_layouts/15/Doc.aspx?sourcedoc=%7B32235514-459E-48F0-AC07-90B1676B0DD7%7D&file=526%20Prioritization%20Scoring%20-%20Feb%202024.xlsx&action=default&mobileredirect=true)

Includes evaluation of user-facing features such as the loading screen during submission, messages post-submission (email and on va.gov), avoid duplicates (rethink when we clear the in-progress form) and the planning, engineering to actually migrate endpoint. Collapsed the ["post-submission" usability research status note](https://docs.google.com/document/d/1qb7mBq0L9sjYOSpLg_PuHgf0JSLGnj12IIucUZXdu5w/edit#heading=h.1ok24b9nnn9) into this

---
## Tasks
- [ ] Desk research on current state of submission notifications and duplicate submissions
- [ ] Desk research with existing documentation, VBA, other teams with knowledge, to understand lifecycle of a claim after submission
- [ ] (if needed) user research to identify issues for users / understand how we might solve them
- [ ] Consider ways to prompt Veterans when a claim is waiting on them to take action
    - [ ] Desk research to understand what is already being done here
- [ ] Create design ticket(s) to address the findings, including
    - [ ] [Help Veterans keep track of their claims post-submission](https://app.zenhub.com/workspaces/disability-experience-63dbdb0a401c4400119d3a44/issues/gh/department-of-veterans-affairs/va.gov-team/78183)
    - [ ] Submission experience
    - [ ] Way to avoid duplicate submissions
    - [ ] Other recommendations found from above steps

## Acceptance Criteria
- [ ] Findings documented and shared with stakeholders
- [ ] Design tickets created to address all issues

**Comment by mengA6** [comment]
@shannonkford tagging you for visibility on this going forward

**Comment by mengA6** [comment]
Need to update this based on [Submission + Post Submission Experience doc](https://docs.google.com/document/d/19H0ZQadc2iPJaK4qasgaLEdnTKbAfrFYH__VhbT_fl8/edit)

**Comment by mengA6** [comment]
Closing as duplicate of other tickets in the [improve submission experience epic](https://app.zenhub.com/workspaces/disability-experience-63dbdb0a401c4400119d3a44/issues/gh/department-of-veterans-affairs/va.gov-team/78179) since we re-worked this including reduced scope.
